test/functional/test_framework/comptool.py

```diff
@@ -360,7 +360,7 @@ def run(self):
                             self.sync_blocks(block.sha256, 1)
                         else:
                             if send_witness:
-                                [c.send_message(msg_witness_block(block)) for c in self.p2p_connections]
+                                [c.send_message(msg_block(block)) for c in self.p2p_connections]
```

Comment:
Also can be simplified.
And perhaps the whole `send_witness` flag can be removed

Reply:
perhaps

Reply:
I did simplify, although I do consider it a sort-of pointless endeavour as we are going to merge 0.17 soonish and comptool goes away in there. Anyway, this file's going to go away but for the time being we work with it.

I did not remove the send_witness flag as this might require me to touch tests which pass it in (going all the way would require that) and that is pointless IMHO as it might just create new merge conflicts for the merge 0.17 task force.

Once 0.17 is merged, or even before during review, we should check that everything works, including witness-only blocks in functional tests, anyhow.